src/main/kotlin/org/dashevo/dpp/statetransition/StateTransition.kt

```diff
@@ -68,17 +67,17 @@ abstract class StateTransition(var signaturePublicKeyId: Int?,
         return Cbor.encode(this.toJSON(skipSignature))
     }
 
-    fun sign(identityPublicKey: IdentityPublicKey, privateKey: String) {
+    /*fun sign(identityPublicKey: IdentityPublicKey, privateKey: String) {
```

Comment:
Can we remove this?

Reply:
Yes

Reply:
Removed here: https://github.com/dashevo/android-dpp/pull/6/commits/fddc9f82d6f720d931b6ec136e0c0ec1843831ba